### PR TITLE
LibGUI: Allow only one CommandPallette at once

### PR DESCRIPTION
**Pull request by sppmacd**

### CODE REVIEW

File: Userland/Libraries/LibGUI/WindowServerConnection.h [hunk 1]
```diff
@@ -58,6 +58,7 @@ class WindowServerConnection final
     virtual void track_mouse_move(Gfx::IntPoint const&) override;
     virtual void ping() override;
 
+    bool m_in_command_pallette { false };
```
Comment (Rummskartoffel) [inline Userland/Libraries/LibGUI/WindowServerConnection.h:61]: Minor spelling error: "palette", not "pallette". Same with the commit message.

Comment (sppmacd) [reply Userland/Libraries/LibGUI/WindowServerConnection.h]: Fixed.

Comment (awesomekling) [reply Userland/Libraries/LibGUI/WindowServerConnection.h]: Commit message still says "pallette"

Comment (sppmacd) [reply Userland/Libraries/LibGUI/WindowServerConnection.h]: Should be OK now.

**Pull request merged**